.github/actions/docker-run/action.yml

```diff
@@ -45,6 +45,15 @@ runs:
       shell: bash
       run: |
         docker pull ghcr.io/${{ github.repository }}/${{ inputs.docker_image }}:${{ env.IMAGE_TAG }}
+    - name: Prepare folders for artifact extracting
+      id: prepare-for-artifacts
+      shell: bash
+      run: |
+        mkdir -p ${{ github.workspace }}/generated
```

Comment:
told @dimitri-tenstorrent that we should get the docker mounting permissions working so that this shouldn't be necessary